From JBeulich@novell.com  Tue Dec 22 11:20:33 2009
From: "Jan Beulich" <JBeulich@novell.com>
Date: Fri, 20 Nov 2009 13:50:53 +0000
Subject: USB: fix section mismatch in early ehci dbgp
To: <gregkh@suse.de>
Cc: <linux-kernel@vger.kernel.org>, <jason.wessel@windriver.com>
Message-ID: <4B06ACCD02000078000211E0@vpn.id2.novell.com>
Content-Disposition: inline


Commit 917778267fbe67703ab7d5c6f0b7a05d4c3df485 removed __init from
ehci_wait_for_port(), but left it in place on ehci_reset_port(), which
is being called from the former function.

Signed-off-by: Jan Beulich <jbeulich@novell.com>
Acked-by: Jason Wessel <jason.wessel@windriver.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/usb/early/ehci-dbgp.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/early/ehci-dbgp.c
+++ b/drivers/usb/early/ehci-dbgp.c
@@ -613,7 +613,7 @@ err:
 }
 EXPORT_SYMBOL_GPL(dbgp_external_startup);
 
-static int __init ehci_reset_port(int port)
+static int ehci_reset_port(int port)
 {
 	u32 portsc;
 	u32 delay_time, delay;
